### ISSUE TITLE
Question about your original paper, maybe an error

### ISSUE THREAD

**Issue by TranSirius** [rrmenon10/ADAPET#20]
Hi,

It's quite insightful to read your paper. However, I am a lit bit confused about the Equation (5). To be more exactly, whether the plus sign should be subtraction sign?

Best regards!

**Comment by rrmenon10** [comment]
Thanks for your interest in our work ! Coming to Equation 5, the sign should be plus there as it is mentioned in the paper. This might seems confusing initially because of the next equation. This is mainly because of the difference in the binary cross entropy loss and the softmax cross-entropy loss. However, if you carefully look at what Equation 5 says, the first term makes sure the right label is assigned a label of 1 in binary cross entropy (equivalently, force the probability of the label to be close to 1) and the second term ensures the wrong label is  assigned the 0 label (equivalently, force the probability of the label to be close to 0). Since we want the probability of the wrong label to go down to zero, we need to reduce that loss, hence the plus sign.
